.circleci/config.yml

```diff
@@ -121,15 +134,62 @@ workflows:
   build_and_deploy:
     jobs:
       - build:
+          filters:
+            branches:
+              ignore:
+                - master
+                - /^support.*/
+                - /^feature\/.*-i18n/
+                - /^release.*/
+      - unit_test:
```

Comment:
for my edification why does the unit step not need filters, but the deploy steps do?

Reply:
talked on hangouts about this!

Reply:
The filters on the build step determine whether or not the workflow (build_and_deploy or build_and_deploy_i18n) will run. The unit test steps depends on the build steps, so only branches that pass the filters on the build steps will be run for the corresponding unit test steps. The deploy steps need filters because the deploy process is different for their associated branches. The deploy step filters are subsets of their corresponding build step filters